### PR TITLE
update backend Readme.md

### PR DESCRIPTION
**Pull request by mhalsubhi**
* update folder structure
* update installation steps
* add seed local database step

#92

### CODE REVIEW

**Review by lelouchB — CHANGES_REQUESTED** [review]
Can you add a few screenshots?

**Comment by mhalsubhi** [comment]
@lelouchB sure I will 👍🏼

File: backend/README.md [hunk 1]
```diff
@@ -52,19 +60,35 @@ backend
...
-4. Create a **.env** folder and add the **Database URL** in it.
+4. Copy `.env.example` file and rename it to `.env`
 
+```bash
+cp .env.example .env
 ```
-DATABASE_URL=mongodb://127.0.0.1:27017/final-space-api
+![cp step](https://i.imgur.com/xbQPLDQ.png)
+
+5. (Optional) seed local database with actual data from api
```
Comment (lelouchB) [inline backend/README.md:86]: Just make this upper case API and we are good to go.

Thanks

Comment (mhalsubhi) [reply backend/README.md]: Done and I add a link to API 👍🏼

**Pull request merged**